weaviate/collections/classes/config_vectorizers.py

```diff
@@ -698,6 +714,47 @@ def text2vec_cohere(
             vectorizeClassName=vectorize_collection_name,
         )
 
+    @staticmethod
+    def multi2vec_cohere(
+        *,
+        model: Optional[Union[CohereModel, str]] = None,
+        truncate: Optional[CohereTruncation] = None,
+        vectorize_collection_name: bool = True,
+        base_url: Optional[AnyHttpUrl] = None,
+        image_fields: Optional[Union[List[str], List[Multi2VecField]]] = None,
+        text_fields: Optional[Union[List[str], List[Multi2VecField]]] = None,
+    ) -> _VectorizerConfigCreate:
+        """Create a `_Multi2VecCohereConfig` object for use when vectorizing using the `multi2vec-cohere` model.
+
+        See the [documentation](https://weaviate.io/developers/weaviate/modules/retriever-vectorizer-modules/text2vec-cohere)
```

Comment:
Same comment re: docstrings; to be updated once docs actually available 

(The others might need updates too, but let's deal with that separately)

Reply:
can you make a PR once the links are updated? :)